### PR TITLE
Only display the "No discounts defined" msg if it is true

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: admin/products_price_manager.php [hunk 1]
```diff
@@ -1088,7 +1088,9 @@ function updateSpecialsNet() {
               </div>
             </div>
           <?php } else { ?>
+            <?php if (sizeof($discount_name) == 0) { ?>
```
Comment (lat9) [inline admin/products_price_manager.php]: Shouldn't we be using `count` instead of `sizeof`?

Comment (proseLA) [reply admin/products_price_manager.php]: we should...

**Comment by scottcwilson** [comment]
`count` is an alias for `sizeof`.  `sizeof` is used throughout this file; `count` is not.

**Comment by lat9** [comment]
Of course it's an alias, but why propagate that older, less desirable usage in new code?

**Comment by Zen4All** [comment]
I agree with @lat9, there has been a move from ```sizeof``` , towards ```count``` lateley

**Comment by drbyte** [comment]
In this case `empty()` is more appropriate.

**Pull request merged**